src/servers/http_server_v2.cc

```diff
@@ -1297,9 +1450,17 @@ HTTPAPIServerV2::HandleInfer(
   if (err == nullptr) {
     std::unique_ptr<InferRequestClass> infer_request(new InferRequestClass(
         req, request_header_protobuf.id(), server_id_, unique_id));
+
+    // Find Inference-Header-Content-Length in header. If missing set to 0
```

Comment:
This is not given when not using binary data... in that case does EVBufferToInput do the right thing?

Reply:
In the case that the header is missing header_length is set to 0. Then in EVBuffer we pick up the evbuffer length and make the entire content the json header.